packages/core/state/interaction/logics.ts

```diff
@@ -489,8 +490,14 @@ const openWithLogic = createLogic({
         } else {
             filesToOpen = await fileSelection.fetchAllDetails();
         }
+
+        // Determine if exePath is local
+        const isLocalExecutable = isAbsolute(exePath);
```

Comment:
I suppose it is likely we will always store the path as absolute for local executables but it does seem potentially buggy with the way certain systems mount apps - I'm pretty sure `executionEnvService` is only used for local apps anyway right? Couldn't we just assume they are always local?